ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -193,48 +215,55 @@ host_monitoring_cron:
 - name: send dnsmasq process count
   minute: "*/5"
   job: ops-runner -f -s 15 -n cspc.openshift.dnsmasq.process.count /usr/bin/cron-send-process-count '^dnsmasq' openshift.dnsmasq.process.count
+# REMOVE_IH: did this ever fail? what does amount of processes tell us besides if it's running or not? 
 
 - name: send openshift-node dnsmasq status every 5 minutes
   minute: "*/5"
   job: ops-runner -f -s 50 -n csosms.openshift.dnsmasq /usr/bin/cron-send-os-dnsmasq-checks
+# REMOVE_IH: can we combine this with the previous check? or use trigger to check for latest of this for readiness?
+
 {# end if dnsmasq #}
 {% endif %}
 
 {% if ( osohm_host_type == 'master' or osohm_host_type == 'infra' ) and osohm_cloud == 'aws' %}
 - name: send ELB status
   minute: "*/15"
   job: ops-runner -f -s 60 -n cses.openshift.aws.elbd.status /usr/bin/cron-send-elb-status
+# REMOVE_IH: should we make this check instead 1 per cluster?
 {% endif %}
 
 {% if osohm_host_type == 'master' or osohm_host_type == 'node' %}
 {# Openshift node checks #}
 - name: send openshift-node process count
   minute: "*/5"
   job: ops-runner -f -s 15 -n cspc.openshift.node.process.count /usr/bin/cron-send-process-count '^/usr/bin/openshift start node' openshift.node.process.count
+# REMOVE_IH: slow down to 10min? as with all process checks we need to do local heals by nudging systemd to deal with process issues
 
 - name: send ovs status data
   minute: "*/5"
   job: ops-runner -f -s 60 -n csos.openshift.master.ovs.status /usr/bin/cron-send-ovs-status
+# REMOVE_IH: need input from watchers on this one  
 
 - name: fix and report on stray OVS rules
   minute: "*/5"
   job: ops-runner -f -s 60 -n cfor.openshift.node.ovs.stray.rules /usr/bin/cron-fix-ovs-rules
+# REMOVE_IH: how stable is ovs nowadays? will slowing this down cause problems for us?
```

Comment:
Decision: investigate if it's still needed. If not, then back off to once an hour.